src/layout.js

```diff
@@ -432,7 +432,7 @@ export function applyStaticLayout(element) {
   if (layout == Layout.NODISPLAY) {
     // CSS defines layout=nodisplay automatically with `display:none`. Thus
     // no additional styling is needed.
-    applyNoDisplayLayout(element);
+    toggle(element, false);
```

Comment:
@honeybadgerdontcare This change should be transparent since we always apply it at runtime (line 337), but probably should update the cache transform regardless.

Reply:
Thanks @choumx, agree it should be transparent. But what do you mean by updating the cache transform? It seems `toggle` with `opt_display` set to `false` still sets `display` to `none`. Am I missing something?

Reply:
Sorry, this is being split across several PRs. The plan is to set the `hidden` attribute instead of `style="display: none"`.

Reply:
Ah. Is the Runtime ready for it being set to `hidden` now? If not what is the timeline?

Reply:
Not yet. Let's wait until it's been in prod for awhile first -- we'll reach out in a few weeks.